frontend/model/contracts/shared/giLodash.js

```diff
@@ -163,6 +163,28 @@ export function deepEqualJSONType (a: any, b: any): boolean {
   return true
 }
 
+export function sortAny (unsorted: any): any {
+  const sort = (unsorted) => {
+    if (!unsorted || typeof unsorted !== 'object') {
+      return unsorted
+    }
+    if (Array.isArray(unsorted)) {
+      return unsorted.map(v => sort(v)).sort()
```

Comment:
I think it's needed because `unsorted` could be an array just like the following.
```
[
  1,
  [2, 4, 1],
  { b: 2, a: 1 },
  undefined,
  0,
  null,
  'any string'
]
```

Reply:
Aha, I got what you mean. Yeah, I don't think it's needed. Nice catch!!!